teku/src/main/java/tech/pegasys/teku/cli/options/P2POptions.java

```diff
@@ -447,12 +460,35 @@ private OptionalInt getP2pLowerBound() {
   }
 
   private OptionalInt getP2pUpperBound() {
-    if (p2pUpperBound.isPresent() && p2pLowerBound.isPresent()) {
-      return p2pLowerBound.getAsInt() > p2pUpperBound.getAsInt() ? p2pLowerBound : p2pUpperBound;
-    }
     return p2pUpperBound;
   }
 
+  private List<String> getStaticPeersList() {
+    List<String> staticPeers = new ArrayList<>(p2pStaticPeers);
```

Comment:
all variables should be final ... I'm surprised the code linting allowed this :)